examples/src/main/java/com/datastax/oss/driver/examples/paging/RandomPagingRestUi.java

```diff
@@ -221,41 +220,38 @@ public void init() {
      * Returns a paginated list of all the videos created by the given user.
      *
      * @param userid the user ID.
-     * @param page the page to request, or {@code null} to get the first page.
+     * @param requestedPageNumber the page to request, or {@code null} to get the first page.
      */
     @GET
     @Path("/{userid}/videos")
     public UserVideosResponse getUserVideos(
-        @PathParam("userid") int userid, @QueryParam("page") Integer page) {
+        @PathParam("userid") int userid, @QueryParam("page") Integer requestedPageNumber) {
 
-      Statement statement = videosByUser.bind(userid).setPageSize(FETCH_SIZE);
+      BoundStatement statement = videosByUser.bind(userid).setPageSize(FETCH_SIZE);
 
-      if (page == null) {
-        page = 1;
+      if (requestedPageNumber == null) {
+        requestedPageNumber = 1;
       }
-      ResultSet rs = pager.skipTo(statement, page);
-
-      List<UserVideo> videos;
-      boolean empty = !rs.iterator().hasNext();
-      if (empty) {
-        videos = Collections.emptyList();
-      } else {
-        int remaining = ITEMS_PER_PAGE;
-        videos = new ArrayList<>(remaining);
-        for (Row row : rs) {
-          UserVideo video =
-              new UserVideo(row.getInt("videoid"), row.getString("title"), row.getInstant("added"));
-          videos.add(video);
-
-          if (--remaining == 0) {
-            break;
-          }
-        }
+      Page<Row> page = pager.getPage(session.execute(statement), requestedPageNumber);
+
+      List<UserVideo> videos = new ArrayList<>(page.getElements().size());
+      for (Row row : page.getElements()) {
+        UserVideo video =
+            new UserVideo(row.getInt("videoid"), row.getString("title"), row.getInstant("added"));
+        videos.add(video);
       }
 
+      // Since we use the RETURN_LAST_PAGE strategy, the actual number could be different from the
```

Comment:
I think that: The comment about `RETURN_LAST_PAGE` is not relevant anymore because the strategy is not present (same for test for this strategy)

Reply:
Good catch.

Reply:
> same for test for this strategy

What do you mean? I didn't find any other text reference to RETURN_LAST_PAGE.

Reply:
I meant this test:
```
  public void should_return_last_page_when_result_finishes_with_empty_frame(int fetchSize) {
```